src/mono/mono/metadata/icall.c

```diff
@@ -7643,8 +7643,15 @@ gchar ***_NSGetEnviron(void);
 G_END_DECLS
 #define environ (*_NSGetEnviron())
 #else
+#ifdef ENABLE_NETCORE
```

Comment:
could you help me understand ENABLE_NETCORE? when would this not be true?

Reply:
(is this mirroring back to mono/mono?)

Reply:
@danmosemsft the runtime source are shared between mono-as-a-vm for .NET 5 and classic mono (mono/mono) and current Xamarin. So we use `ENABLE_NETCORE` to make .NET 5 only related changes

Reply:
Just to clarify, any PRs touching the runtime C/C++ sources are mirrored. We have some netcore-specific source/header files, the icall defines being a good example, but most of the code is still shared so you'll see the `ENABLE_NETCORE` define used throughout.

Reply:
gotcha. I had assumed such changes all started in mono/mono and were mirrored here.